unit1-wrap-up/activity-mind-map.md

```diff
@@ -0,0 +1,13 @@
+# Unit 1 - Reflection Activity
+
+Wow! We've learned so much in the last 5 weeks. Let's take time to reflect on our learning and make connections.
+
+We will use a technique called a mind map. A mind map is a diagram that can represent tasks, ideas, concepts, etc. and the connections between them. In our case, we can include anything from our learnings from Unit 1 -- whether it be python programming concepts, study skills, learnings about how to be in community, or anything else.
+
+These are a few tools our team recommends: 
+* [Google Slide](https://www.google.com/slides/about/)
+* [Mind Meister](https://www.mindmeister.com/)
+
+and here is a [more exhaustive list](https://zapier.com/blog/best-mind-mapping-software/) of mind mapping software.
```

Comment:
Why not just pick a few from this list? A number of these aren't free

Reply:
I tried to pick some, and I didn't love anything